modules/core/core-api/src/main/java/com/enonic/xp/project/ProjectConstants.java

```diff
@@ -29,6 +29,10 @@
 
     public static final String PROJECT_ACCESS_LEVEL_CONTRIBUTOR_PROPERTY = "contributor";
 
+    public static final String PROJECT_READ_ACCESS_JSON_PROPERTY_NAME = "readAccess";
```

Comment:
Should not be here?

Reply:
hmm, why not?

Reply:
Because the only use of it is in admin-impl
So, it is internal  stuff
Same applies to PROJECT_PERMISSIONS_JSON_PROPERTY_NAME

So move these constraints to admin-impl, I guess.

Reply:
ah, mixed up the location of ProjectConstants. agree.